scylladb/python-driver
# docs: moved latest_version to conf.py

**Related issue:** https://github.com/scylladb/sphinx-scylladb-theme/issues/87

The latest stable version will be defined in ``conf.py`` instead of using the GitHub Actions file.
With the new setting``smv_rename_latest_version``, we can rename the URL of the latest stable version.

@fruch With this change,  you won't have to maintain a stable tag anymore. Instead, the variable ``smv_rename_latest_version`` will rename the URL of the latest version to ``latest`` automatically.

Every time you release a new stable version, you will need to update the setting``smv_latest_version`` in ``conf.py``.

##  How to test this PR

1. Run ``make multiversionpreview``. When you open https://0.0.0.0:5500,  it should redirect you to the latest version.
2. The URL for the version ``3.24.1-scylla`` should be https://0.0.0.0:5500/stable
